modules/imaging_browser/php/queryengine.class.inc

```diff
@@ -137,22 +329,16 @@ class QueryEngine extends \LORIS\Data\Query\SQLQueryEngine
                              WHEN s.Scan_Done='N' THEN false
                              ELSE NULL END";
         }
-        if ($item instanceof LocationDictionaryItem) {
-            $modality = $item->getModality();
 
-            $this->addTable('LEFT JOIN session s ON (s.CandID=c.CandID)');
-            $this->addTable("LEFT JOIN files ON (s.ID=files.SessionID)");
+        /* @phan-suppress-next-line PhanUndeclaredMethod */
```

Comment:
Don't suppress phan errors.

If you're getting "UndeclaredMethod" errors it's likely because you moved the call outside of the instanceof check that ensured it was declared.